app/models/geographical_area.rb

```diff
@@ -3,46 +3,51 @@
 class GeographicalArea
   include ApiEntity
 
-  collection_path '/geographical_areas/countries'
+  collection_path "/geographical_areas/countries"
 
   attr_accessor :id, :description, :geographical_area_id
 
   has_many :children_geographical_areas, class_name: 'GeographicalArea'
 
   def self.countries
-    excluded_geographical_area_ids = %w[GB]
+    excluded_geographical_area_ids =
+      if TradeTariffFrontend::ServiceChooser.xi?
+        []
+      else
+        %w[GB]
+      end
```

Comment:
This could be:

```ruby
excluded_geographical_area_ids = TradeTariffFrontend::ServiceChooser.xi? ? [] : %w[GB]
```

Reply:
I actually really don't like ternary operators.